modules/services/gpg-agent.nix

```diff
@@ -129,6 +129,22 @@ in
           configuration file.
         '';
       };
+
+      pinentryFlavor = mkOption {
+        type = types.nullOr (types.enum pkgs.pinentry.flavors);
+        example = "gnome3";
+        default = "gtk2";
+        description = ''
+          Which pinentry interface to use. If not null, the path to the
+          pinentry binary will be passed to gpg-agent via commandline and
+          thus overrides the pinentry option in gpg-agent.conf in the user's
+          home directory.
+          If not set at all, it'll pick qt, unlike the similar NixOS
```

Comment:
It says `qt` here but default is `gtk2`?

Reply:
Haha yes, sorry. I experimented with different defaults to see which one seemed to work best